scrunch/datasets.py

```diff
@@ -761,15 +806,19 @@ def graph(self, _):
     def get(self):
         # Returns the synchronized hierarchical order graph.
         if self._sync:
-            ds_state = self.ds.resource.session.get(
-                self.ds.resource.self + 'state/'
-            ).payload
-            if self._revision is None:
-                self._revision = ds_state.body.revision
-            elif self._revision != ds_state.body.revision:
-                # There's a new dataset revision. Reload the
-                # hierarchical order.
-                self._revision = ds_state.body.revision
+            if hasattr(self.resource, 'variables'):
+                ds_state = self.resource.session.get(
+                    self.resource.self + 'state/'
+                ).payload
```

Comment:
We don't concatenate strings, if you need the /state/ just follow `.resource.state`